File: Modul 6/SOAL 3/PYTHON/PRAK603.py
nilai1, nilai2 = map(int, input().split())

if (nilai1 == nilai2):
    elemen1 = list(map(int, input().split()))
    elemen2 = list(map(int, input().split()))
    baris1 = elemen1
    baris2 = elemen2
    for i in range(nilai1):
        baris1.append(elemen1)
    for i in range(nilai2):
        baris2.append(elemen2)
    print() 
    for i in range(nilai1):
        hasil = baris1[i] * baris2[i]
        print(hasil, end = " ")
else:
    print("\nJumlah tidak sama")

File: Modul 6/SOAL 3/C/PRAK603.c
#include <stdio.h>

int main() {
    int nilai1, nilai2, elemen1, elemen2, i;

    scanf("%d %d", &nilai1, &nilai2);

    int baris1[nilai1], baris2[nilai2];

    if (nilai1 == nilai2) {
        for (i = 0; i < nilai1; i++) {
            scanf("%d", &elemen1);
            baris1[i] = elemen1;
        }
        for (i = 0; i < nilai2; i++) {
            scanf("%d", &elemen2);
            baris2[i] = elemen2;
        } printf("\n");

        for (i = 0; i < nilai1; i++) {
            int hasil = baris1[i] * baris2[i];
            printf("%d ", hasil);
        }
    } else {
        printf("\nJumlah tidak sama");
    } 
    return 0;
}